packages/synthetix-governance/contracts/storage/DebtShareStorage.sol

```diff
@@ -0,0 +1,32 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "../interfaces/IDebtShare.sol";
+import "@synthetixio/core-contracts/contracts/utils/SetUtil.sol";
+
+contract DebtShareStorage {
```

Comment:
SNX specific debt share storage is now in a separate namespace than generic election module storage. I believe that this is important for future updates, and for the election storage to not get messy.